Update README.md (#733)
w3c-fedid/FedCM: README.md

Developer documentation: [Federated Credential Management (FedCM) API](https://developer.mozilla.org/en-US/docs/Web/API/FedCM_API)

Editor's draft of the spec [here](https://w3c-fedid.github.io/FedCM/)

## Introduction

The Federated Credential Management (FedCM) API provides a modern, privacy-preserving

> standard for federated identity on the web. It enables a more secure and intuitive

Reviewer: ```suggestion
standard for federated identity on the web. It enables a more secure and intuitive
```

"secure" -- i'm trying to think if "more secure" is defensible ... in what ways do we expect it to be "more secure"?

user experience for signing in to websites (Relying Parties) with accounts from a
trusted Identity Provider, all through a browser-mediated and standardized flow.

In an era of increasing focus on user privacy and data control, the FedCM API offers
a forward-looking solution to the known privacy and usability challenges of

> traditional federated login systems. By shifting control to the browser, it creates

Reviewer: ```suggestion
traditional federated login systems. By introducing the browser as an active participant, it creates
```

> a more transparent and consistent authentication process. This ensures that users

Reviewer: ```suggestion
a more interoperable and consistent authentication process. This ensures that users
```

have a clearer understanding and more explicit control over what information is
being shared and with whom.

> For developers, FedCM offers a durable and streamlined API for implementing

Reviewer: ```suggestion
For websites, FedCM offers a durable and efficient API for implementing
```

Reviewer: maybe it would be useful to different "websites" with "identity providers", rather than talk to "developers" as a whole?

federated sign-in that is not dependent on the shifting landscape of browser
tracking technologies. It represents a proactive step towards building a more
trustworthy and user-centric web by improving the fundamental mechanics of how
we manage identity online.

> 

Reviewer: For identity providers, FedCM offers increased distribution via new novel access points (e.g. allowing multiple idps to co-exist in the same account chooser and allowing users to choose their own), that wouldn't be possible unless the browser was involved.

      interface to FedCM which will need to be utilized.

 * If you're a Relying Party (i.e. website) and would like to test the changes out
   we'd appreciate feedback, you have a couple of options:

>    1. Use [HOWTO.md](explorations/HOWTO-chrome.md) to setup a fake IDP which can serve

Author: Sure we already have that link above anyways so changed this one
